docker-entrypoint.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -eo pipefail
+shopt -s nullglob
+OVERPASS_META=${OVERPASS_META:-no}
+OVERPASS_MODE=${OVERPASS_MODE:-clone}
+OVERPASS_COMPRESSION=${OVERPASS_COMPRESSION:-gz}
+
+if [ ! -d /db/db ] ; then
+    if [ "$OVERPASS_MODE" = "clone" ]; then
+        if [[ "$OVERPASS_META" == "attic" ]] ; then
+            META="--keep-attic"
+        else
+            META="--meta=$OVERPASS_META"
+        fi
+        mkdir -p /db/db \
+        && /app/bin/download_clone.sh --db-dir=/db/db --source=http://dev.overpass-api.de/api_drolbr/ $META "--compresion_method=$OVERPASS_COMPRESSION" \
```

Comment:
download_clone.sh just downloads existing files with a predefined compression setting, i.e.  there's not much point to provide a compression method parameter, as it will not have any impact.